### ISSUE TITLE
Atomic Free: Restrict VideoPress access

### ISSUE THREAD

**Issue by dereksmart** [Automattic/wp-calypso#51593]
VideoPress is already blocked on the backend, but we need to reflect this properly in the Calypso UI. 

### What plan tier is the site on?
Free

### Where are you seeing the issue?
Media > Videos 

#### What I expected on the Atomic site
![image](https://user-images.githubusercontent.com/7129409/113173784-bd22a700-9217-11eb-88cb-b4898826f1c2.png)

#### What actually happened
![image](https://user-images.githubusercontent.com/7129409/113173881-d75c8500-9217-11eb-8205-e249158663f8.png)

**Comment by tjcafferkey** [comment]
It looks like this has since been fixed and is no longer reproducible.

**Comment by tjcafferkey** [comment]
>It looks like this has since been fixed and is no longer reproducible.

I take that back

**Comment by obenland** [comment]
Let's check `upload_file_types` to make sure that video extensions are not part of that.

**Comment by DustyReagan** [comment]
This seems to be testing correctly for me. @tjcafferkey I see this issue was marked as closed, then re-opened, is there a way to reproduce this that I missed?

Atomic Free

![atomic-free](https://user-images.githubusercontent.com/140841/151882202-e00f2332-4016-47c7-a969-98623ba8f5be.png)

Atomic Personal

![atomic-personal](https://user-images.githubusercontent.com/140841/151882205-decaf294-d6ee-489d-b776-d7f420be5865.png)

Atomic Premium

![atomic-premium](https://user-images.githubusercontent.com/140841/151882197-9ac97233-0b4a-46f5-bc6e-5e2a0f2a3e30.png)

**Comment by mmtr** [comment]
I tested a bunch of scenarios, and everything seems to be working as expected on Calypso. However, I found some discrepancies in WP Admin.

Page | Simple (Free) | Atomic (Free)
--- | --- | ---
`wp-admin/post-new.php` | After inserting a Video block, both "Upload" and "Media library" buttons are disabled, so I'm unable to upload any video file. <img width="682" alt="Screen Shot 2022-02-08 at 12 29 35" src="https://user-images.githubusercontent.com/1233880/152979059-1a46c23c-9146-4910-88ab-dbb9d8980244.png"> | After inserting a Video block, both "Upload" and "Media library" buttons are enabled, so I can upload video files. <img width="652" alt="Screen Shot 2022-02-08 at 12 30 11" src="https://user-images.githubusercontent.com/1233880/152979244-5e9895c8-dab5-4799-8fc2-d12524ef6037.png">
`wp-admin/upload.php` | The allowed file types don't include videos and I cannot upload them. <img width="988" alt="Screen Shot 2022-02-08 at 12 30 53" src="https://user-images.githubusercontent.com/1233880/152979629-a32a7ce0-95f4-4893-b24a-c23ecab4f6d9.png"> | There are no restricted file types, so I can upload video files. <img width="989" alt="Screen Shot 2022-02-08 at 12 31 01" src="https://user-images.githubusercontent.com/1233880/152979752-8a7b051b-a480-4de4-911f-30acc6c9f97f.png">
